app/code/Magento/InventoryGroupedProductAdminUi/etc/module.xml

```diff
@@ -6,5 +6,9 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_InventoryGroupedProductAdminUi" setup_version="1.0.0" />
+    <module name="Magento_InventoryGroupedProductAdminUi" setup_version="1.0.0">
+        <sequence>
+            <module name="Magento_InventoryGroupedProduct"/>
```

Comment:
Why do we need sequence declaration? Looks like in our case doesn't matter the order of modules installing